posts/2022-02-16-space-sentry-challenge.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Space Sentry Challenge: A Java Space Odyssey Game with flying saucers, Lego, Arduino, and Java microservices, built on Open Liberty"
```

Comment:
replace the colons (:) in the values in the metadata section with dashes (-). For some reason, colons cause the metadata block to fail and the post won't display on the draft site. Colon is ok in the actual title, just metadata needs edit

 "Space Sentry Challenge- A Java Space Odyssey Game...